### ISSUE TITLE
Expectancy lookup by `word` fails for words that are not all lowercase

### ISSUE THREAD

**Issue by whoopsedesy** [sasansom/sedes#84]
In the [generalized-expectancy](https://github.com/sasansom/sedes/tree/526941f65bf88efb82db7d3bfbfe632e4f0d40ff) branch (#83), if you use `word` as one of the variables by which expectancy is computed, and the word contains a letter that is not lowercase (`word.lower() != word`), then the expectancy lookup in tei2html-generated HTML doesn't work. The embedded `EXPECTANCY` table in the HTML uses lowercase keys, while the input to the lookup does not lowercase the lookup key.

E.g.

```
$ src/expectancy --by sedes/word corpus/*.csv > expectancy.sedes_word.csv
$ src/tei2html --by sedes/word corpus/aratus.xml expectancy.sedes_word.csv > "sedes-web/aratus.html"
```

The `EXPECTANCY` entry is lowercase:

```
["διὸς", new Map([
  ["10", {x: 62, z: -0.97579118}],
  ["2", {x: 58, z: -1.0128046}],
  ["2.5", {x: 85, z: -0.76296419}],
  ["4", {x: 60, z: -0.99429788}],
  ["4.5", {x: 12, z: -1.4384585}],
  ["6", {x: 24, z: -1.3274184}],
  ["6.5", {x: 294, z: +1.1709854}],
  ["7", {x: 1, z: -1.5402454}],
  ["8", {x: 124, z: -0.40208365}],
])],
```

But the contents of the `<span class="word">` is capitalized:

```
<span class="sedes s2"><span id="book-1-line-1-word-2" class="word z-17" data-wordno="2" data-lemma="ζεύς" data-sedes="2" data-shape="⏑⏑" title="lemma=ζεύς sedes=2 x=58 Σx=720 z=−1.0128046 shape=⏑⏑">Διὸς</span></span>
```

The [lookup](https://github.com/sasansom/sedes/blob/526941f65bf88efb82db7d3bfbfe632e4f0d40ff/src/tei2html#L931) just uses the `textContent` of the `span`.

**Comment by whoopsedesy** [comment]
The lowercasing happens [in `hexameter.analyze_line_metrical`](https://github.com/sasansom/sedes/blob/526941f65bf88efb82db7d3bfbfe632e4f0d40ff/src/hexameter/scan.py#L378).

**Comment by whoopsedesy** [comment]
> keep on lowercasing the input as we do now, and only change the lookup in tei2html to also apply the lowercase transformation.

In 6936e07efc001765da9399e08ddd3fa78906cbb2 I have taken this option, having the HTML emitted by tei2html lowercase words before looking up their expectancy.

It still may be possible to modify tei2csv to retain letter case in the `word` column, but that will require answers to questions like the ones in https://github.com/sasansom/sedes/issues/84#issuecomment-1776581672.

**Comment by whoopsedesy** [comment]
This was fixed in #83. I rebased the commit that fixed the problem into the initial commit that created a need for it.
